### PR TITLE
install-cli add windows

### PR DESCRIPTION
**Pull request by feigu1984**
windows install to $HOME/ioctl.exe

### CODE REVIEW

File: install-cli.sh [hunk 1]
```diff
@@ -77,11 +77,12 @@ findGoBinDirectory() {
             echo "Installation could not determine your \$GOPATH."
             exit 1
         fi
-        if [ -z "$GOBIN" ]; then
-            GOBIN=$(echo "${EFFECTIVE_GOPATH%%:*}/bin" | sed s#//*#/#g)
+        if [ -z "$GOROOT" ]; then
```
Comment (raullenchai) [inline install-cli.sh]: why check on GOROOT is better than check on GOBIN?

Comment (raullenchai) [reply install-cli.sh]: windows has no GOBIN?

Comment (raullenchai) [reply install-cli.sh]: I prefer to completely get rid of GO env. We should install in three fixed dirs for macos, linux and windows.

File: install-cli.sh [hunk 2]
```diff
@@ -77,11 +77,12 @@ findGoBinDirectory() {
...
+            #GOBIN=$(echo "${EFFECTIVE_GOPATH%%:*}/bin" | sed s#//*#/#g)
+	    GOBIN=$GOROOT/bin
         fi
-        if [ ! -d "$GOBIN" ]; then
```
Comment (feigu1984) [inline install-cli.sh]: windows  but golang installed

File: install-cli.sh [hunk 3]
```diff
@@ -77,11 +78,12 @@ findGoBinDirectory() {
             echo "Installation could not determine your \$GOPATH."
             exit 1
         fi
-        if [ -z "$GOBIN" ]; then
-            GOBIN=$(echo "${EFFECTIVE_GOPATH%%:*}/bin" | sed s#//*#/#g)
+        if [ -z "$GOROOT" ]; then
```
Comment (raullenchai) [inline install-cli.sh]: can remove findGoBinDirectory

File: install-cli.sh [hunk 4]
```diff
@@ -128,10 +130,9 @@ initArch
 initOS
 
 # determine install directory if required
```
Comment (raullenchai) [inline install-cli.sh]: remove if not needed; update the comment in file header

**Pull request merged**